gridcapa-swe-runner-app/src/test/java/com/farao_community/farao/swe/runner/app/voltage/VoltageResultMapperTest.java

```diff
@@ -32,7 +40,7 @@ void mapVoltageResultTestCheckListAndState() {
     @Test
     void mapVoltageResultTestCheckListElememnts() {
         result.getConstraintElements().stream().forEach(voltResult -> {
-            if (voltResult.getInstant() == Instant.CURATIVE) {
+            if (voltResult.getInstant().equals("CURATIVE")) {
```

Comment:
this is not null safe either:
- use string utils .equals(), or
- "CURATIVE".equals(voltResult.getInstant()

:) regards